deploy/dev/Jenkinsfile

```diff
@@ -1,4 +1,7 @@
 /* Requires the Docker Pipeline plugin */
+/*
+    Angus: Out of curiosity (not a criticism), what made you choose Jenkins over GitHub Actions?
```

Comment:
Thanks for your question. When I initially began creating a pipeline at work, Jenkins was the first tool I came across. Its comprehensiveness and user-friendly nature immediately struck me. While I haven't personally experimented with GitHub Actions, I really appreciate its simplicity and effectiveness, particularly for smaller projects such as this.

Jenkins, however, has proven its worth by enabling me to construct a robust and efficient pipeline with parallel stages for quick building and deployment of applications. I believe the choice between the two really boils down to the specific needs and scale of the project at hand.